ipi/engine/normalmodes.py

```diff
@@ -499,7 +534,244 @@ def get_dynm3(self):
                     dm3[k, a] = self.beads.m3[k, a] * self.o_nm_factor[k]
         return dm3
 
+    def Evaluate_EkN(self, N, k):
```

Comment:
I was wondering if it would be cleaner to create a separate file called ipi.utils.exchange.py (or a better sounding name) and import the functions from there ? May not be a bad idea since that could be more lines about fermions coming in the future.

Reply:
Hi, I think that is a good idea. I tried moving everything to ipi.utils.exchange as you suggested. 
Can you have a look? Thanks.